### DISCUSSION TITLE
Different background for Flex group

### DISCUSSION THREAD

**Discussion by Redflashx12** [fltk-rs/fltk-rs#1468] [Q&A] [answered]
Hello,

I wanted to ask if it's possible to change the background for a single group of widgets to be able to differentiate them from the background and other widget groups, like how it's typically done in cards in many UIs, e.g. like here (gray-ish background and white cards):
![image](https://github.com/fltk-rs/fltk-rs/assets/43482469/6d212659-d8c5-4fd1-a7b5-474280df05ce)

I took your example of tabs.rs, adjusted it slightly and made it work for the normal "group::Group" by adding a frame first with the desired size and color, but it didn't work using e.g. either the Flex's or Group's "set_color" method. Is there any way to do this for Flex? Or maybe also not as hard-coded for Groups? Currently, it looks like this:
![image](https://github.com/fltk-rs/fltk-rs/assets/43482469/c53ab58a-08a1-4f18-a935-e54be63fa0ba)
On the left is the Flex group, and on the right is the "normal" Group.

My code:
```
use fltk::{
    app,
    button::*,
    group::{Flex, Tabs},
    input::Input,
    menu::{Choice, MenuButton},
    output::Output,
    prelude::{GroupExt, InputExt, MenuExt, WidgetBase, WidgetExt, WindowExt},
    window::Window,
};
use fltk::enums::{Color, FrameType};
use fltk::frame::Frame;
use fltk::group::Group;

fn draw_gallery() {
    let mut tab = Tabs::default_fill();

    let mut grp1 = Flex::default_fill().with_label("Tab1\t\t").row();
    let mut col = Flex::default().column();
    col.set_color(Color::White); // <------ This doesn't do anything
    grp1.fixed(&col, 160);
    col.set_pad(32);
    col.set_margin(32);
    let _but1 = Button::default().with_label("Button");
    let _but2 = RoundButton::default().with_label("Round");
    col.end();

    let mut group = Group::default_fill();
    grp1.fixed(&group, 160);

    let mut frame = Frame::new(-8, 16, 80, 400, "");
    frame.set_color(Color::White);
    frame.set_frame(FrameType::RFlatBox);
    let _but3 = CheckButton::default().with_size(64, 32).with_pos(0, 32).with_label("Check");
    let _but4 = LightButton::default_fill().with_size(64, 32).below_of(&_but3, 32).with_label("Light");

    group.end();
    grp1.end();

    let grp2 = Flex::default_fill().with_label("Tab2\t\t").row();
    grp2.end();
    tab.end();
    tab.auto_layout();
}

fn main() {
    let app = app::App::default().with_scheme(app::Scheme::Gtk);
    app::background(221, 221, 221);

    let mut wind = Window::default()
        .with_size(500, 450)
        .with_label("Tabs")
        .center_screen();

    draw_gallery();

    wind.make_resizable(true);
    wind.end();
    wind.show();

    app.run().unwrap();
}
```

**Comment by MoAlyousef** [comment, marked as answer]
Hello

It's because some widgets by default have a NoBox as a FrameType, which would require you to change it to something like a FlatBox to get an opaque color.
```rust
col.set_frame(FrameType::FlatBox);
```

In this way, it similar to the Frame widget, which here you set to have an RFlatBox.
